backend/src/main/java/hu/flowacademy/meteo/controller/AirDataController.java

```diff
@@ -0,0 +1,26 @@
+package hu.flowacademy.meteo.controller;
+
+import hu.flowacademy.meteo.dto.AirDataDto;
+import hu.flowacademy.meteo.dto.BetweenDatesDto;
+import hu.flowacademy.meteo.dto.MeasurmentDto;
+import hu.flowacademy.meteo.service.MeasurmentService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequiredArgsConstructor
+public class AirDataController {
+
+    private final MeasurmentService measurmentService;
+
+    @PostMapping("air")
```

Comment:
Ez egy get request, attol hogy a body is at van passzolva, hacsak nem enged get-be bodyt rakni akkor nyugodtan lehet GetMapping.

Reply:
igen ezért látt átírva postra mert gettel sajnos nem müködött a request body-s változat.

Reply:
Getnél nincs request body, csak query string. Viszont esetünkben lekérésről van szó, szóval én GET-et használnék @RequestParam-al.

Reply:
> igen ez ezért lett átírva postra mert gettel sajnos nem müködött a request body-s változat.

Erdekes pedig a post/get kozott a lenyegi kulonbseg, hogy mit is csinalnak, a post nem idempotens es resource-t hoz letre. Get-nek kene lennie es annak hogy body-ja vna nem kene kizarni, hogy get legyen.

Reply:
> > igen ez ezért lett átírva postra mert gettel sajnos nem müködött a request body-s változat.
> 
> Erdekes pedig a post/get kozott a lenyegi kulonbseg, hogy mit is csinalnak, a post nem idempotens es resource-t hoz letre. Get-nek kene lennie es annak hogy body-ja vna nem kene kizarni, hogy get legyen.

Technikailag lehet, csak nem érdemes, mert a GET eredményét elvileg csak a request uri határozhatja meg és nem jól definiált a viselkedés a specifikáció szerint, ha van body: https://stackoverflow.com/questions/978061/http-get-with-request-body

Reply:
A felhasznalas viszont mar eltendal ettol, itt is irja, hogy mar ezeket a kiteteleket eltavolitottak a specifikaciobol es jellemzoen backward compatibilitas miatt kerdese ennek a megoldasnak a hasznalata.

Reply:
Ok, csak a böngészők olyannyira nem követték ezt le, hogy nem tudnak GET-ben bodyt küldeni: https://stackoverflow.com/questions/46404051/send-object-with-axios-get-request

"According to the documentation and the spec XMLHttpRequest ignores the body of the request in case the method is GET. If you perform a request in Chrome/Electron with XMLHttpRequest and you try to put a json body in the send method this just gets ignored.

Using fetch which is the modern replacement for XMLHtppRequest also seems to fail in Chrome/Electron."

A konklúzió: ha GET, akkor query string, ha POST, akkor lehet bodyban. Mivel ez egy lekérés, ezért szerintem a REST szemantikával szembemegy, hogy POST legyen.

Reply:
Igen, ebben a formaban teljesen egyet ertek a dologgal.